packages/gasket-utils/lib/require-with-install.js

```diff
@@ -59,4 +59,23 @@ async function requireWithInstall(dependency, gasket) {
   return require(resolve(dependency, resolveOptions));
 }
 
+/**
+ * requireWithInstall - load devDependency request programmatically when needed
+ * @param {string|string[]} dependencyRequest The require'ed dep/s needed
+ * @param {Gasket} gasket Gasket instance
+ * @returns {object|object[]} module or list of modules
+ */
+async function requireWithInstall(dependencyRequest, gasket) {
+  if (!Array.isArray(dependencyRequest)) {
+    const resolvedDependency = await requireDependency(dependencyRequest, gasket);
+    return resolvedDependency;
+  }
+  const resolvedDependencyList = [];
+  for (const dependency of dependencyRequest) {
+    const resolvedDependency = await requireDependency(dependency, gasket);
+    resolvedDependencyList.push(resolvedDependency);
+  }
```

Comment:
What we need with the multi-dependency support is for the underlying `npm` call to be executed once with the list of modules to install. i.e. we should see
```
npm install my-package @scoped/package --no-save
```
and not
```
npm install my-package --no-save
npm install @scoped/package --no-save
```

The problem being, when the second install happens, npm will remove the previously installed but not saved package from node_modules.